src/adapter/src/coord/sequencer/cluster.rs

```diff
@@ -697,9 +684,9 @@ impl Coordinator {
         Ok(ExecuteResponse::AlteredObject(ObjectType::Cluster))
     }
 
-    async fn sequence_alter_cluster_managed_to_managed(
+    pub async fn sequence_alter_cluster_managed_to_managed(
         &mut self,
-        session: &Session,
+        session: Option<&Session>,
```

Comment:
One way to fix this is instead of taking an `Option<&Session>` can we change this to take a `ConnectionId`. Then we can pass the `SYSTEM_CONN_ID` which should fix the issue with audit log. Although looking through the code we might panic in catalog transact because I'm not sure the Coordinator knows about `SYSTEM_CONN_ID`.

If we do panic then we can fix this in a follow-up PR

Reply:
I've tried this (2f45e1caa9257f6cc697b691792ba96b4ac87ab5) but it's indeed panicking:
```
thread 'coordinator' panicked at src/adapter/src/coord/ddl.rs:484:58:
connection must exist
```

Reply:
Ahhh darn, do you mind filing an issue so we can followup?

Reply:
Created an issue: https://github.com/MaterializeInc/materialize/issues/26615